lancelot.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Raizel",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-13.0-20230911-lancelot-v9.8.zip",
+			"download": "https://sourceforge.net/projects/rom-builders/files/lancelot/crDroidAndroid-13.0-20230911-lancelot-v9.8.zip/download",
```

Comment:
Use generated download url, not your own

Reply:
Rom-builders is wrong
 Not sure where you took that

Reply:
Fixed